openedx/frontend-app-learning
# TNL-7126 BD-29 Visual Updates to Learning Sequence

This PR updates some minor visual issues with the Learning Sequence:
- Changing `fa-film` icon to `fa-video`
- Adding `react-helmet` to update the page title

**JIRA tickets**:
- [OSPR-4725](https://openedx.atlassian.net/browse/OSPR-4725)
- [TNL-7126](https://openedx.atlassian.net/browse/TNL-7126)
- BD-29

**Dependencies**: None

**Screenshots**:
`fa-video` icon:
![faVideoIcon](https://user-images.githubusercontent.com/829980/84975094-cd690a00-b14e-11ea-846e-7b48be5ae1c4.png)

Title Change:
![titleChange](https://user-images.githubusercontent.com/829980/84977757-5a16c680-b155-11ea-8c3a-9d2fb7814a56.png)

**Merge deadline**: None

**Testing instructions**:

1. Pull the `open-craft:patrick/BB-2569-tnl-7126-minor-visual-updates-to-learner-sequen` branch locally and setup your local LMS.
2. Run `npm install && npm start` in your `frontend-app-learning` folder
3. Login to Studio and edit the demo course to add a new video unit.
4. Navigate to the video unit on your new learning sequence experience
5. Observe the unit icon, and title changes

**Author notes and concerns**:

1. There may be further changes for the sequence bar if needed
2. Initially, a `CourseLicense` component was added to display licensing, but due to the metadata api not containing the necessary information to display the course licensing, this has been removed from this PR.

**Reviewers**
- [x] @edx/teaching-and-learning
